sdk/cosmos/azure-cosmos/azure/cosmos/aio/database.py

```diff
@@ -144,15 +145,15 @@ async def read(self, **kwargs):
 
     @distributed_trace_async
     async def create_container(
-        self,
-        id,  # type: str  # pylint: disable=redefined-builtin
-        partition_key,  # type: Any
-        indexing_policy=None,  # type: Optional[Dict[str, Any]]
-        default_ttl=None,  # type: Optional[int]
-        offer_throughput=None,  # type: Optional[int]
-        unique_key_policy=None,  # type: Optional[Dict[str, Any]]
-        conflict_resolution_policy=None,  # type: Optional[Dict[str, Any]]
-        **kwargs  # type: Any
+            self,
+            id,  # type: str  # pylint: disable=redefined-builtin
+            partition_key,  # type: Any
+            indexing_policy=None,  # type: Optional[Dict[str, Any]]
+            default_ttl=None,  # type: Optional[int]
+            offer_throughput=None,  # type: Optional[int]
+            unique_key_policy=None,  # type: Optional[Dict[str, Any]]
+            conflict_resolution_policy=None,  # type: Optional[Dict[str, Any]]
+            **kwargs  # type: Any
```

Comment:
The original indentation here looks correct. What's the reason for this change?

Reply:
Pycharm did this on its own to "format the file correctly"